tensorflow/core/kernels/fused_batch_norm_op.cc

```diff
@@ -1264,6 +1264,20 @@ class FusedBatchNormOpBase : public OpKernel {
         context, estimated_variance.dims() == 1,
         errors::InvalidArgument("estimated_variance must be 1-dimensional",
                                 estimated_variance.shape().DebugString()));
+    bool use_reshape = (x.dims() == 5);
+    auto x_shape = x.shape();
+    TensorShape dest_shape;
+    if (use_reshape) {
+      const int64 in_batch = GetTensorDim(x, tensor_format_, 'N');
+      int64 in_planes = GetTensorDim(x, tensor_format_, '0');
+      int64 in_rows = GetTensorDim(x, tensor_format_, '1');
+      int64 in_cols = GetTensorDim(x, tensor_format_, '2');
+      const int64 in_depth = GetTensorDim(x, tensor_format_, 'C');
+      dest_shape = ShapeFromFormat(
+          tensor_format_, in_batch, {{in_planes, in_rows * in_cols}}, in_depth);
+      CHECK(x.CopyFrom(x, dest_shape));
```

Comment:
Can these `CHECK`s be converted to `OP_REQUIRES`? Currently that is triggering one of our presubmits recommending against using CHECK.

Reply:
Done. PTAL.